src/Login/Login.js

```diff
@@ -14,22 +14,46 @@ class Login extends Component {
     }
   }
 
-  submitLogin() {
+  submitLogin = (e) => {
+    e.preventDefault();
     const userInfo = { email: this.state.email, password: this.state.password };
-    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/login', {
-      method: 'POST',
-      headers: {"Content-Type": "application/json"},
-      body: JSON.stringify(userInfo)
-    })
-    .then(response => response.json())
-    .then(data => this.setState({
-      email: data.user.email,
-      id: data.user.id,
-      name: data.user.name
-    }))
+    if (this.state.email === 'rick@turing.io' && this.state.password === 'asdf123') {
+      fetch('https://rancid-tomatillos.herokuapp.com/api/v2/login', {
+        method: 'POST',
+        headers: {"Content-Type": "application/json"},
+        body: JSON.stringify(userInfo)
+      })
+      .then(response => response.json())
+      .then(data => this.setState({
+        email: data.user.email,
+        id: data.user.id,
+        name: data.user.name
+      }))
+      .then(data => this.createUser())
+      .then(data => this.goToHome())
```

Comment:
It works without issues, but I'm wondering if calling these helper functions in chained .then() statements is the best way to go.